src/pages/leaderboard.tsx

```diff
@@ -99,6 +108,21 @@ function Leaderboard() {
     }
   }, [formatDate]);
 
+  useEffect(() => {
+    async function getTopMemes() {
+      setTopMemesLoading(true);
+      const response = await fetch(
+        `https://evening-anchorage-43225.herokuapp.com/museum/memes/?created_at__gte=&created_at__lte=${toDate}&created_at=&created_at__gt=${fromDate}&created_at__lt=`
```

Comment:
Over time we should probably look at putting e.g. the api url into the .env variables.